source/clear-linux/get-started/virtual-machine-install/vmware-player-2.rst

```diff
@@ -0,0 +1,469 @@
+
+#.  Select the newly created VM and click `Play virtual machine`.  
+
+    |vmware-player-08|
+
+    Figure 8: VMware Workstation 14 Player - Power on virtual machine
+
+#.  Follow the :ref:`bare-metal-install` (starting at the `Install Clear Linux 
+    on your target system` section) guide to complete the installation of 
+    |CL|.
+
+#.  After the installation is complete, follow the |CL| instruction to reboot it.  
+    This will restart the installer again.  Follow the next section.  
+
+Reconfigure the VM settings to boot the newly installed |CL|
+************************************************************
+
+#.  Enable the mouse pointer.
+
+    - Press `<CTRL>` + `<ALT>` to enable the mouse pointer.
+
+#.  Disconnect the CD/DVD to stop it from booting the installer ISO again.
+    
+    * Click `Player`.
+    * Go to `Removable Devices` > `CD/DVD (IDE)` > `Settings`.
+
+      |vmware-player-09|
+
+      Figure 9: VMware Workstation 14 Player - Edit CD/DVD settings
+
+    * Under `Device status`, uncheck `Connected` and `Connect at power on`.
+    * Click `OK`.
+
+      |vmware-player-10|
+
+      Figure 10: VMware Workstation 14 Player - Disconnect CD/DVD
+
+#.  Power off the VM.
+
+    * Click `Player`.
+    * Go to `Power` and select `Shut Down Guest`.
+
+      |vmware-player-11|
+
+      Figure 11: VMware Workstation 14 Player - Power off virtual machine
+
+Enable UEFI boot support
+************************
+
+|CL| needs UEFI support in order to boot.  Enable it by appending the following 
+the end of the VM's `.vmx` file (located in the directory where the VM resides
+see Note below):
+
+  .. code-block:: console
+
+    firmware = "efi"
+
+.. note::
+
+  VMware VM files are typically located in:
+
+  * Linux distros: `/home/username/vmware`
+  * Windows: `C:/\Users/\username/\Documents/\Virtual Machines`
+
+Power on the virtual machine
+****************************
+
+After configuring the settings above, power on the virtual machine.  
+
+Method 2: Boot a ready-made VMware |CL| image
+=============================================
+
+The general process for booting a ready-made VMware |CL| image is as follows 
+(with expanded details below):
+
+* Create a new VM and configure its base settings
+* Attach the ready-made VMware |CL| image
+* Enable EFI boot support
+* Power on the VM
+
+Create a new VM and configure its base settings
+***********************************************
+
+#.  Start the `VMware Workstation Player` app.
+#.  On the home screen:
+
+    * Click `Create a New Virtual Machine`.
+
+      |vmware-player-01|
+
+      Figure 12: VMware Workstation 14 Player - Create a new virtual machine
+   
+#.  On the `Welcome to the New Virtual Machine Wizard` step:
+
+    * Select `I will install the operating system later`.
+    * Click `Next`.
+
+      |vmware-player-13|
+
+      Figure 13: VMware Workstation 14 Player - Select install operating system 
+      later  
+
+#.  On the `Select a Guest Operating System` step:
```

Comment:
Same feedback for these steps. Simply have the title of the step be the instruction:
#. Select a guest operating system: